.gitignore

```diff
@@ -3,3 +3,143 @@ bin
 .bake_cache
 .vscode
 gcov
+/build/ZERO_CHECK.vcxproj.filters
```

Comment:
I think this was accidentally checked in? If you'd like to ignore everything in the build directory, you can just add "build" to the `.gitignore` file.